package.json

```diff
@@ -9,30 +9,23 @@
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --configFile babel.config.js --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:storybook": "build-storybook -o storybook_dist",
-    "ci:packages": "npm ci && lerna exec -- npm ci",
-    "changeScanner": "node scripts/changeScanner",
-    "bumpPackages": "node scripts/bumpPackages/cli",
     "bumpChangelogs": "node scripts/bumpChangelogs/cli",
-    "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
+    "bumpPackages": "node scripts/bumpPackages/cli",
+    "changeScanner": "node scripts/changeScanner",
+    "ci:packages": "npm ci && lerna exec -- npm ci",
     "deploy-storybook": "gh-pages -d storybook_dist",
-    "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; sed -i \"\" -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "install:packages": "npm i && lerna clean --yes && lerna bootstrap --hoist",
-    "install:packages:link": "npm i && lerna clean --yes && lerna bootstrap --hoist --force-local",
-    "postinstall": "npm run build",
-    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi",
+    "postshrinkwrap": "if test -z $CI; then \n ./scripts/forcePackageLockHttps.sh \n fi",
     "preinstall": "sh scripts/enforceVersions.sh",
+    "postinstall": "npm run build",
     "publish": "npm run build && node scripts/publish",
+    "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
     "storybook": "NODE_ENV=development start-storybook -p 8180 -c .storybook",
     "storybook:ie11": "export FIX_STORYBOOK_IE11=true && npm run storybook",
-    "test": "npm run test:lint && npm run test:lint:css && npm run test:unit && npm run test:packages",
-    "test:chromatic": "chromatic test run --storybook-build-dir=storybook_dist || true",
-    "test:packages": "lerna run test",
-    "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts",
-    "test:lint:css": "stylelint 'packages/**/*.js' 'packages/**/*.jsx' 'scripts/**/*.js'",
-    "test:unit": "npm run build && jest --verbose --coverage",
-    "test:unit:watch": "npm run build && jest --verbose --coverage --watch",
     "talos": "node scripts/talos/cli",
```

Comment:
We can remove talos from here, and just call it from the make file, it shouldnt really we called locally :P